lib/core/elastic_connector_actions.rb

```diff
@@ -235,12 +235,99 @@ def ensure_connectors_index_exists
           :properties => {
             :api_key_id => { :type => :keyword },
             :configuration => { :type => :object },
-            :error => { :type => :text },
+            :description => { :type => :text },
```

Comment:
TIL that this function exists. Sorry I missed this for filtering rules. I don't love that we're maintaining this in yet _another_ place, but this isn't the time to change that.